Composer/packages/adaptive-form/src/components/sharedStyles.ts

```diff
@@ -19,3 +19,5 @@ export const focusBorder = css`
     }
   }
 `;
+
+export const sharedFieldIconStyles = { backgroundColor: '#F3F2F1', width: '44px', lineHeight: '28px', fontSize: 12 };
```

Comment:
is that color one of the NeutralColors provided by Fabric?
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/neutrals
(Grey 20)